### ISSUE TITLE
Shall we move this repo to docker/dockerpty?

### ISSUE THREAD

**Issue by d11wtq** [d11wtq/dockerpty#31]
Given the lack of involvement from me of late, and the fact this project now has multiple collaborators, does it make more sense to move this repository across to the docker organization?

**Comment by aanand** [comment]
Makes sense to me, particularly if Compose is the primary/only consumer of it. @bfirsh, @samalba?

**Comment by d11wtq** [comment]
There are definitely other consumers, so I would avoid merging it into the one project. I know [Harpoon](https://github.com/realestate-com-au/harpoon) uses it.

**Comment by aanand** [comment]
Oh, I don't mean to suggest we roll it into Compose - just that if you're happy to hand it over to github.com/docker, I think it's a good idea.

**Comment by d11wtq** [comment]
How should we do this? Change ownership? Do we need to coordinate?

**Comment by aanand** [comment]
Ping @samalba, who's got the GitHub power I believe. Sam - what would be involved in moving this repo to the docker org?
